collabnext/openalex/institutions.py

```diff
@@ -1,6 +1,34 @@
 from pyalex import Institution, Institutions
+import json
+import os
+import scripts.fetch_custom_institutions as fetch_custom_institutions
 
 
-def get_institutions() -> list[Institution]:
-    # Get 5 random institutions for now
-    return [Institutions().random() for _ in range(5)]
+def get_institutions(institutions_file_path: str = "observable/docs/data/institutions.json") -> list[Institution]:
+    institutions = []
+    
+    # Load institutions from JSON file
+    try:        
+        institutions = json.load(open(institutions_file_path))        
+    except Exception as e:
+        print("\nError loading institutions from JSON file", institutions_file_path, ":", e, "\n")
```

Comment:
stdout is used for outputting data in an observable framework data loader so it's not possible to use print.  This should work (haven't tried, need to import sys of course):

```suggestion
        print("\nError loading institutions from JSON file", institutions_file_path, ":", e, "\n", file=sys.stderr)
```

Reply:
Updated in the new commit